app/src/components/market/sections/market_buy/scalar_market_buy.tsx

```diff
@@ -207,12 +207,18 @@ export const ScalarMarketBuy = (props: Props) => {
   const potentialProfit = tradedShares.isZero() ? new BigNumber(0) : tradedShares.sub(amount)
 
   const currentBalance = `${formatBigNumber(collateralBalance, collateral.decimals, 5)}`
-  const feeFormatted = `${formatNumber(formatBigNumber(feePaid.mul(-1), collateral.decimals))} ${collateral.symbol}`
-  const baseCostFormatted = `${formatNumber(formatBigNumber(baseCost, collateral.decimals))} ${collateral.symbol}`
+  const feeFormatted = `${formatNumber(formatBigNumber(feePaid.mul(-1), collateral.decimals, collateral.decimals))} ${
+    collateral.symbol
+  }`
+  const baseCostFormatted = `${formatNumber(formatBigNumber(baseCost, collateral.decimals, collateral.decimals))} ${
+    collateral.symbol
+  }`
   const potentialProfitFormatted = potentialProfit.gt(Zero)
-    ? `${formatNumber(Number(formatBigNumber(potentialProfit, collateral.decimals)).toString())} ${collateral.symbol}`
+    ? `${formatNumber(Number(formatBigNumber(potentialProfit, collateral.decimals, collateral.decimals)).toString())} ${
```

Comment:
I think this is from a past PR, but the `Number().toString()` part is redundant. Instead it should just be: 
`formatNumber(formatBigNumber(potentialProfit, collateral.decimals, collateral.decimals)))`